README.md

```diff
@@ -34,6 +34,12 @@ Questions? Comments? Start by perusing our [GitHub discussions](https://github.c
 
 Please see [the documentation](https://nautobot.readthedocs.io/en/stable/installation/) for instructions on installing Nautobot.
 
+## Application Stack
+
+Below is a simplified overview of the Nautobot application stack for reference:
+
+![Application stack diagram](./docs/media/nautobot_application_stack_high_level.png "Application stack diagram")
```

Comment:
`README.md` (and *only* README.md) gets rendered to PyPI when we do a new release. Relative paths to images from this file will be broken links on PyPI - like the other image links already in README.md, this needs to be an absolute https link, either to readthedocs.org or to raw.githubusercontent.com (the latter is what the other images in this file are using, so probably best to be consistent).